tests/unit/Logger.js

```diff
@@ -176,7 +180,7 @@ describe('WereLogs Logger is usable:', () => {
     it('Uses the additional fields as expected', done => {
         const dummyLogger = new DummyLogger();
         const logger = new Logger('test');
```

Comment:
Since you're actually directly importing the Logger constructor (from logger.js), which is good in unit-testing; you also need to provide the configuration instance to the logger constructor; according to the design I suggested. I see that here it works because you instanciate a new Config instance each time a Logger is created; but that is the opposite of what I aimed with my suggestion.